### ISSUE TITLE
Monster Hunter AFS_DATA files contain extra bytes

### ISSUE THREAD

**Issue by 2Tie** [MaikelChan/AFSPacker#7]
Some, but not all, files unpacked from the JP or US releases of Monster Hunter for the PS2 appear to have two extra bytes at the start of data and four blank bytes at the end of data, and possibly extra bytes throughout - this is easily observable to occur with the PNG files such as TITLE1.PNG, where the unpacked files are invalid (magic not at start, bytes after END chunk), and occurs as well with the .apx files. The offsets and size listed for the file entries are used correctly, there just seems to be some extra bytes included in the file's data.

Other files, such as JP's game.bin and related .bin files at the top of the .AFS listing, seem to not have this quirk at first glance and have their magic at the top of their files. Linked is the JP AFS_DATA; the US game.bin has the same problems i was encountering with the JP .png files.

http://dongs.biz/storage/AFS_DATA.AFS

**Comment by MaikelChan** [comment]
Hi!

I just checked your AFS file. It is how you say it is, some files are apparently fine and some others have some extra bytes at the beginning. I also saw it in .htm files. But this is how they are actually stored. It's not rare to find weird variations or modifications of common formats in games. So even if it's annoying, in this case you would have to handle these variations: remove the bytes, edit the file, add the bytes again. It would be out of the scope of this program to be able to know any possible variation of any format out there.

If you have any other question don't hesitate to ask.
